tests/test_namespace.py

```diff
@@ -0,0 +1,58 @@
+from contextlib import contextmanager
+
+from cms.api import create_page
+from django.test import Client
+from django.utils.encoding import force_str
+
+from .base import BaseTest
+from .test_utils.models import CustomUser as User
+
+try:
+    from knocker.signals import pause_knocks
+except ImportError:
+
+    @contextmanager
+    def pause_knocks(obj):
+        yield
+
+
+class AdminTest(BaseTest):
+    def setUp(self):
+        User.objects.create_superuser("test_admin", "admin@example.com", "Password123")
+
+    def add_blog_config(self, page, namespace, id):
+        self.client = Client()
+        self.client.login(username="test_admin", password="Password123")
```

Comment:
you can use `self._admin_user_username`, `self._admin_user_email`  and `self._admin_user_password` to make the login, an admin user is automatically created and you can drop the `setUp` method above

Reply:
Thank you for the suggestion! Makes live easier.